### DISCUSSION TITLE
How to change MyT password ?

### DISCUSSION THREAD

**Discussion by molusk** [DurgNomis-drol/ha_toyota#116] [Q&A] [answered]
Hi,

I had to change my MyT password and will have to do it again soon (have an issue open with Toyota Support, they need to access my account...). I did not find the way to update my password in Home Assistant.

I would rather not delete the integration and reinstall it to keep all the configuration around (automantion, dashboards...). Is there any other way ?

Maybe it is more a Home Assistant issue than ha_toyota... is that case, just tell me and I will ask this elsewhere.

Thank's in advance.

**Comment by DurgNomis-drol** [comment, marked as answer]
Just got to the integration page and try to set it up again. It will just replace the integration without changing anything other then the password

**Reply by molusk** [reply]
Thank's a lot !
